webhook_server_container/libs/github_api.py

```diff
@@ -758,12 +821,42 @@ def process_pull_request_webhook_data(self):
                 ):
                     self._remove_label(label=_label_name)
 
+            if self.jira_track_pr:
+                _story_label = [_label for _label in self.pull_request.labels if _label.name.startswith(JIRA_STR)]
+                if _story_label:
+                    _story_key = _story_label[0].name.split(":")[-1]
+                    self.get_jira_conn()
+                    if not self.jira_conn:
+                        self.app.logger.error(f"{self.log_prefix} Jira connection not found")
```

Comment:
move log to `get_jira_conn` ?

Reply:
No, get_jira_connection have ignore_exception so it wil ldo nothing.